src/azure-cli/azure/cli/command_modules/sqlvm/_params.py

```diff
@@ -168,6 +175,9 @@ def load_arguments(self, _):
                    options_list=['--image-sku'],
                    help='SQL image sku.',
                    arg_type=get_enum_type(SqlImageSku))
+        c.argument('sql_image_offer',
+                   options_list=['--image-offer'],
+                   help='SQL image offer. Examples include SQL2008R2-WS2008R2, SQL2008R2-WS2008, SQL2008-WS2008, SQL2008-WS2008R2.')
```

Comment:
Where do these values come from?  It seems like this might need to use the `populator_commands` kwarg in `help.py`.

Reply:
These were just examples of possible combinations. I reduced the amount of examples just to give an idea of what the pattern should look like.

Reply:
Is it purely pattern based? Is there an endpoint that provides actual supported values?

Reply:
No there is no endpoint that provides the supported values unfortunately. This parameter (image-offer) is only required if customer provides sql-mgmt-type as NoAgent, which is a minimal portion of customers (what we call End of Support Customers that have WS 2008).